### DISCUSSION TITLE
Support for large files (> 4GB)

### DISCUSSION THREAD

**Discussion by corsair400r** [ente-io/ente#583] [Enhancements]
upload videos larger than 4GB, maybe 50gb will be better in the 4k era would be better for the future of this app as I have already a lot of videos larger than 4GB that I cannot backup to enter and is a shame

**Comment by umrath** [comment]
Actually, this is a surprising limitation and almost a deal breaker for me.
I’m currently migrating my media from Apple Photos and this is a constant problem.

A 4 GB file size limit is really not state of the art anymore.

**Reply by vishnukvmd** [reply]
Just an update - we're actively working on [video streaming](https://help.ente.io/photos/faq/video-streaming) to remove this limitation.

**Reply by NickDiMucci** [reply]
Firstly, the progress the team continues to make on Ente is great and continues to maintain my support 💚 💯 🎉  

With video streaming now being in v1, are there any other blockers to removing the 4GB limitation?

**Reply by vishnukvmd** [reply]
Video streaming is still in beta, we'll roll it out on desktop shortly so you can backfill streams for your existing videos. General availability will follow.

We've already started working on removing the size limitation. These changes should be live in the coming months.

**Comment by brog-io** [comment]
This has been added in the new release, the new limit is 10gb

**Reply by constantins2001** [reply]
Still a bit small, I have some 4k Videos that reach 15GB. Is support for larger files planned?
